study-builder/studies/osteo/patches/prequal-updates.conf

```diff
@@ -87,7 +87,7 @@
         }
       ],
       "validation":{
-        "varName": "osteo_prequal_validation_self_international",
+        "varName": """!user.studies["CMI-OSTEO"].forms["PREQUAL"].questions["SELF_COUNTRY"].answers.hasAnyOption("US", "CA", "PR", "GU", "VI", "MP", "AS")""",
         "newVal": """Currently, the OS project is open only to patients in the United States or Canada. If you also live or are treated in the United States or Canada, please reach out to us at <a href="mailto:info@osproject.org" class="Link">info@osproject.org</a>"""
       }
```

Comment:
Why is varName changed to some PEX expression?
As i understand now it will try to find variable with name of that expression and i doubt that it will be able to find it,